test/jdk/java/foreign/TestLayouts.java

```diff
@@ -166,6 +168,51 @@ public void testStructOverflow() {
                                                 MemoryLayout.sequenceLayout(Long.MAX_VALUE, JAVA_BYTE)));
     }
 
+    @Test
+    public void testPadding() {
+        var padding = MemoryLayout.paddingLayout(8);
+        assertEquals(padding.byteAlignment(), 1);
+    }
+
+    @Test
+    public void testPaddingInStruct() {
+        var padding = MemoryLayout.paddingLayout(8);
+        var struct = MemoryLayout.structLayout(padding);
+        assertEquals(struct.byteAlignment(), 1);
+    }
+
+    @Test
+    public void testPaddingIllegalBitSize() {
+        for (long bitSize : List.of(-1L, 1L, 7L)) {
+            try {
+                MemoryLayout.paddingLayout(bitSize);
+                fail("bitSize cannot be " + bitSize);
+            } catch (IllegalArgumentException ignore) {
+                // Happy path
+            }
+        }
+    }
+
+    @Test
+    public void testPaddingZeroBitSize() {
+        MemoryLayout.paddingLayout(0);
+    }
+
+    @Test
+    public void testNullMember() {
+        var illegalLayouts = new MemoryLayout[]{JAVA_INT, null, JAVA_INT};
```

Comment:
this is covered by TestNulls. That test will try to pass null values to all API methods accepting arrays, or collections, and will do so in an automatic fashion.